src/test/e2e/30_config_file_test.go

```diff
@@ -164,3 +165,58 @@ func configFileDefaultTests(t *testing.T) {
 
 	os.Unsetenv("ZARF_CONFIG")
 }
+
+func configFileTestsWithFlag(t *testing.T, dir, path, configPath string) {
```

Comment:
There is a lot of copying and pasting in this test, it may be better to simply check one variable. This function could then test just the --config-path flag and the other function will test that the config file works as intended.

Reply:
just added this in the latest commit